runway/templates/k8s-flux-repo/flux.tf/main.tf

```diff
@@ -0,0 +1,468 @@
+
+terraform {
+  backend "s3" {
+    key = "eks-flux.tfstate"
+  }
+}
+
+locals {
+  cluster_name = "k8s-${terraform.workspace}"
+  flux_repository = "flux-${terraform.workspace}"
+  flux_namespace = "flux"
+  flux_service_account = "flux"
+  flux_version = "1.15.0"
+}
+
+data "aws_eks_cluster" "cluster" {
+  name = "${local.cluster_name}"
+}
+
+data "aws_eks_cluster_auth" "cluster_auth" {
+  name = "${data.aws_eks_cluster.cluster.id}"
+}
+
+data "aws_ssm_parameter" "oidc_iam_provider_cluster_url" {
+  name = "/${local.cluster_name}/oidc-iam-provider-cluster-url"
+}
+data "aws_ssm_parameter" "oidc_iam_provider_cluster_arn" {
+  name = "/${local.cluster_name}/oidc-iam-provider-cluster-arn"
+}
+
+provider "kubernetes" {
+  host = "${data.aws_eks_cluster.cluster.endpoint}"
+  cluster_ca_certificate = "${base64decode(data.aws_eks_cluster.cluster.certificate_authority.0.data)}"
+  token = "${data.aws_eks_cluster_auth.cluster_auth.token}"
+  load_config_file = false
+  version = "1.9"
```

Comment:
Should sync this up with the version (& associated comment on why it's pinned back to 1.10.0) with the eks-base.tf